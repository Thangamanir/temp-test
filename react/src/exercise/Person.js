import React, { Component } from "react";

function Person(props) {
  const name = "Hello World!";
  return <h3>{name}</h3>;
}

export default Person;
