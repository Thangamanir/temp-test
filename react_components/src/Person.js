import React, { Component } from "react";

const Person = props => {
  //Enter code for Person Component
  //return <h3>I am a Person</h3>;
};

export default Person;
