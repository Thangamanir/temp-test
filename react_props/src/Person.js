import React from "react";

function Person  ( props ){
  //Enter code for Person Component
  return <h3>Hello {props.name}</h3>;
};

export default Person;
