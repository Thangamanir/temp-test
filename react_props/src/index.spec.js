import React from "react";
import {create} from "react-test-renderer";
import Person from "./Person";

describe("The Person component", () => {
  it("Testing Person Component", () => {
    const component = create(<Person/>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
