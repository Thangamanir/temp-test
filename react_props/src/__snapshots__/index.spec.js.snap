// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Person component Testing Person Component 1`] = `
<h3>
  Hello 
</h3>
`;
